src/pages/lessons/fundamentals/eth-intro-part-2.mdx

```diff
@@ -0,0 +1,279 @@
+
+## Using an account
+
+This is an important takeaway: **the only way to affect a change on the blockchain is via a transaction** and every transaction must be signed by an account. Accounts can initiate transactions which transfer ether, deploy smart contracts, or interact with contracts to do things like mint new tokens. Let's briefly explore each.
+
+### Transferring ether
+
+Recall that `EthereumTesterProvider` enables a test environment seeded with accounts and fake ether. Let's start by viewing some test accounts and an account balance:
+
+```python
+In [1]: w3 = Web3(Web3.EthereumTesterProvider())
+
+In [2]: w3.eth.accounts
+Out[2]: ['0x7E5F4552091A69125d5DfCb7b8C2659029395Bdf',
+ '0x2B5AD5c4795c026514f8317c7a215E218DcCD6cF',
+ '0x6813Eb9362372EEF6200f3b1dbC3f819671cBA69', ...]
+
+In [3]: acct_one = w3.eth.accounts[0]
+
+In [4]: w3.eth.get_balance(acct_one)
+Out[4]: 1000000000000000000000000
+```
+
+Next, we'll introduce a new account:
+
+```python
+In [5]: acct_two = w3.eth.account.create()
+
+In [6]: acct_two.address
+Out[6]: '0x2FF32Bcc040e98EBa3c0ae2d8ad9C451a78d3E24'
+
+In [7]: acct_two.key
+Out[7]: HexBytes('0x02af55504048265...f70e9965a3505ea')
+```
+
+Then send one of those fake ether over to the new account:
+
+```python
+In [8]: tx_hash = w3.eth.send_transaction({
+      'from': acct_one,
+      'to': acct_two.address,
+      'value': Web3.toWei(1, 'ether')
+   })
+```
+
+This transaction will execute immediately, but some important details are hidden from view. web3.py is smart enough to know that `EthereumTesterProvider` is managing `acct_one` and that we're using a test environment. For our convenience, `acct_one` is "unlocked," meaning transactions from the account are approved (signed) by default.
+
+So, what does a transaction look like if not from an unlocked account? To find out, let's send some ether from `acct_two` – an account not managed by `EthereumTesterProvider`. This more manual process takes three steps: 1) specify the transaction details, 2) sign the transaction, then 3) broadcast the transaction to the network.
+
+```python
+# 1) manually build a transaction
+In [9]: tx = {
+          'to': acct_one,
+          'value': 10000000,
+          'gas': 21000,
+          'gasPrice': w3.eth.get_block('pending')['baseFeePerGas'],
+          'nonce': 0
+        }
+
+# 2) sign the transaction with the sender's private key
+In[10]: signed = w3.eth.account.sign_transaction(tx, acct_two.key)
+
+# 3) send the "raw" transaction
+In[11]: tx_hash = w3.eth.send_raw_transaction(signed.rawTransaction)
+```
+
+Let's break that down. Step 1 defines a Python dictionary with the required transaction fields. We briefly learned about gas (transaction fees) in Part 1, but nonce may be new territory. In Ethereum, a **nonce** is simply the transaction count of the account. The Ethereum protocol keeps track of this value to prevent double-spending.
+
```

Comment:
Would you mind using this as a very small learning moment to educate non-Python-heads what a dictionary is in Python, e.g. in brackets after '...required transaction fields (A dictionary, or `dict` is a Python type similar to an object in other programming languages).' And then consolidate it three paragraphs further down with '...a from value is missing from the `tx` dict'. Just an idea.

Reply:
Lovely job!!